Index: speechsep/lightning.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/speechsep/lightning.py b/speechsep/lightning.py
--- a/speechsep/lightning.py	(revision 4879d3388c08b36f376de9326d841203f44b7e69)
+++ b/speechsep/lightning.py	(date 1667570738972)
@@ -41,32 +41,35 @@
     from torch.utils.data import DataLoader
     import os
 
-    train = False
+    train = True
 
     is_hpc = "LSF_ENVDIR" in os.environ
-    use_gpu = is_hpc and torch.cuda.is_available()
+    use_gpu = torch.cuda.is_available()
     test_checkpoint_path = "data/lightning_logs/version_4/checkpoints/epoch=19-step=1280.ckpt"
 
     if use_gpu:
-        dataloader_args = {"batch_size": 16, "num_workers": 4, "persistent_workers": True}
+        dataloader_args = {"batch_size": 1, "num_workers": 4, "persistent_workers": True}
         trainer_args = {
-            "max_epochs": 20,
+            "max_epochs": 100,
             "accelerator": "gpu",
             "devices": 1,
             "auto_select_gpus": True,
         }
     else:
-        dataloader_args = {"batch_size": 4}
+        dataloader_args = {"batch_size": 1}
         trainer_args = {"max_epochs": 2}
 
     model = LitDemucs()
 
     if train:
         train_dataloader = DataLoader(
-            SinusoidDataset(2048, example_length=1, extend_to_valid=True), **dataloader_args
+            SinusoidDataset(2, example_length=1, extend_to_valid=True), **dataloader_args
         )
 
-        trainer = pl.Trainer(log_every_n_steps=32, default_root_dir="data/", **trainer_args)
+        trainer = pl.Trainer(log_every_n_steps=32,
+                             default_root_dir="data/",
+                             **trainer_args,
+                             enable_checkpointing=False)
         trainer.fit(model=model, train_dataloaders=train_dataloader)
     else:
         dataloader = DataLoader(SinusoidDataset(1, example_length=1, pad_to_valid=True))
